src/NuGet.Services.AzureSearch/Auxiliary2AzureSearch/Auxiliary2AzureSearchCommand.cs

```diff
@@ -144,6 +144,10 @@ private async Task<bool> PushIndexChangesAsync()
             _logger.LogInformation("Removing invalid IDs and versions from the new data.");
             CleanDownloadData(newData);
 
+            _logger.LogInformation("Overriding download count data.");
+            var overrideData = await _auxiliaryFileClient.LoadDownloadOverridesAsync();
+            overrideData.Update(newData);
```

Comment:
This is changing the download count that appears in search. I don't think this is intended. Ideally we boost but still show the "real" number.

Reply:
Yup, I found this out during testing (this is still a draft! 😅). I'll add unit tests for this case.

Reply:
Fixed this issue and added tests